Makefile

```
@@ -46,18 +45,16 @@ test-compile:
 	fi
 	go test -c $(TEST) $(TESTARGS)
 
-website:
-	ifeq (,$(wildcard $(GOPATH)/src/$(WEBSITE_REPO)))
-		echo "$(WEBSITE_REPO) not found in your GOPATH (necessary for layouts and assets), get-ting..."
-		git clone https://$(WEBSITE_REPO) $(GOPATH)/src/$(WEBSITE_REPO)
-	endif
-		@$(MAKE) -C $(GOPATH)/src/$(WEBSITE_REPO) website-provider PROVIDER_PATH=$(shell pwd) PROVIDER_NAME=$(PKG_NAME)
-
-website-test:
-	ifeq (,$(wildcard $(GOPATH)/src/$(WEBSITE_REPO)))
-		echo "$(WEBSITE_REPO) not found in your GOPATH (necessary for layouts and assets), get-ting..."
-		git clone https://$(WEBSITE_REPO) $(GOPATH)/src/$(WEBSITE_REPO)
-	endif
-		@$(MAKE) -C $(GOPATH)/src/$(WEBSITE_REPO) website-provider-test PROVIDER_PATH=$(shell pwd) PROVIDER_NAME=$(PKG_NAME)
-
-.PHONY: build test testacc vet fmt fmtcheck errcheck vendor-status test-compile website website-test
+website-repo: clean
+	git clone https://github.com/hashicorp/$(WEBSITE_REPO) website-repo
```

As above, not sure why we are hardcoding hashicorp here
```suggestion
	git clone https://github.com/$(WEBSITE_REPO) website-repo
```